cpapp_backend/cpapp/api/document/views.py

```
@@ -0,0 +1,306 @@
+                return Response(
+                    {'error': f'Error processing PAN card: {str(e)}'}, 
+                    status=status.HTTP_500_INTERNAL_SERVER_ERROR
+                )
+                
+            finally:
+                # Clean up temporary file
+                if tmp_path and os.path.exists(tmp_path):
+                    try:
+                        os.unlink(tmp_path)
+                        logger.info(f"Cleaned up temporary PAN card file: {tmp_path}")
+                    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)